app/models/service_ansible_playbook.rb

```diff
@@ -45,6 +45,12 @@ def postprocess(action)
     delete_inventory(action) unless use_default_inventory?(hosts)
   end
 
+  def jobs
```

Comment:
We should have one Provision and one Retirement job, but for Reconfigure that we may support in the future, it can have multiple jobs.
I am hesitated too about this matter. We don't have clear answer whether we want to support Reconfigure. If so, even the existing `job` method needs to return an array.